easybuild/easyconfigs/c/CmdStanR/CmdStanR-0.5.2-foss-2022a-R-4.2.1.eb

```diff
@@ -0,0 +1,32 @@
+easyblock = 'RPackage'
+
+name = 'CmdStanR'
+version = '0.5.2'
+versionsuffix = '-R-%(rver)s'
+
+homepage = 'https://mc-stan.org/cmdstanr'
+description = "CmdStanR is a lightweight interface to Stan for R users"
+
+toolchain = {'name': 'foss', 'version': '2022a'}
+
+
+sources = [{
+	'filename': 'cmdStanR-%(version)s.tar.gz',
+	'git_config': {
```

Comment:
`git_config` should only be used when either Git metadata must be available (which requires enabling the `keep_git_dir` option), or when Git submodules are involved (which requires enabling the `recursive` option).

Since neither is apparently required, the source tarball served by GitHub can be downloaded directly via `https://github.com/stan-dev/cmdstanr/archive` (as is done in the existing `CmdStanR-0.5.0-foss-2021b-R-4.1.2.eb` easyconfig).